### DISCUSSION TITLE
Re-enable actions / other repo settings

### DISCUSSION THREAD

**Discussion by ian-h-chamberlain** [rust3ds/citro3d-rs#17] [General]
Unfortunately, it seems like this repo lost some of its settings when it was transferred to the `rust3ds` organization. @Meziu can you see if it's possible to add me either as a maintainer or admin to this repo so I can re-enable it?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository#changing-permissions-for-a-team-or-person

It might also be worth considering setting the whole @rust3ds/active group as maintainers on this and `ctru-rs`/`cargo-3ds` in case there are other repo settings needed, but I don't think it matters too much (mostly, I want to be able to turn on CI and release lifecycle stuff).

**Comment by Meziu** [comment]
I’ve given maintainer status to the @rust3ds/active team 👍

**Reply by ian-h-chamberlain** [reply]
Thanks! I can see the `Settings` links on `ctru-rs` and `cargo-3ds`, but not this repo, maybe something is different about it since it was transferred in?

Still it does looks like I would be able to add actions, and I misremembered having CI on this repo (I think I was looking at `ctru-rs` when I said it), so maybe I'll be able to add it without any extra steps anyway. I should have more time to dig into this in another week or so.
